space/Dockerfile.space

```diff
@@ -20,19 +19,16 @@ RUN yarn install --network-timeout 500000
 
 COPY --from=builder /app/out/full/ .
 COPY turbo.json turbo.json
+COPY ./space/.env .env
```

Comment:
Can you explain this line?

Reply:
You're correct, this line doesn't make sense, as `turbo prune` imports everything and would be importing the `.env` from the project folder as well. Removed it ✅